doc/rippled-example.cfg

```diff
@@ -374,12 +374,6 @@
 #       When set, activates the autoconnect feature. This maintains outgoing
 #       connections using PeerFinder's "Outgoing Connection Strategy."
 #
-#   http_handshake = 0 | 1
-#
-#       When set, outgoing peer connections will handshaking using a HTTP
-#       request instead of the legacy TMHello protocol buffers message.
-#       Incoming peer connections have their handshakes detected automatically.
```

Comment:
Do we know if anybody was using `http_handshake = 1` in the wild?

Per the "Tx Changes" label, I forgot this setting was available. Perhaps this isn't breaking, but would require anybody who wants to stay on 0.27.0 to set this flag.